### PR TITLE
Allow force push to tempo and opentelemetry operator repos

### PR DESCRIPTION
**Pull request by pavolloffay**

### CODE REVIEW

File: core-services/prow/02_config/openshift/grafana-tempo-operator/_prowconfig.yaml
Comment (IshwarKanse) [inline core-services/prow/02_config/openshift/grafana-tempo-operator/_prowconfig.yaml:8]: We can remove protect: false when unmanaged is defined.

**Review by frzifus — APPROVED** [review]

**Comment by droslean** [comment]
/hold

Temporary holding to unblock the queue

**Pull request merged**
